src/pages/documentation/adobe-campaign-standard/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## May 25, 2023
+
+### Android Campaign Standard 2.0.4
+
+* Update the AEPCore dependency to version 2.2.0 to resolve a WebView transparency issue and fix message frequency rules for ACS in-app messages.
```

Comment:
ACS = Adobe Campaign Standard?

I think we should probably make it "... for Campaign Standard in-app messages." instead.

Reply:
you are correct, it is Adobe Campaign Standard. good point, will fix it.